howto/cluster/appliance.md

```diff
@@ -0,0 +1,47 @@
+If you're running the Anbox Cloud Appliance and you want to extend the available container capacity, you can join additional machines to the appliance.
+
+The following instructions describe how to add a machine. Also see [Multi-node support for the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/capacity-planning/17765#clustering-anbox-cloud).
+
+## Allow internal network communication
+
+To allow communication between the different machines, you must configure the internal network that the machines use to allow incoming traffic.
+
+How to do this depends on your network configuration. On AWS, for example, you can modify the security group attached to the instance to allow all incoming traffic on the internal subnet that it sits on:
+
+![AWS security group](images/appliance_multi_node_aws_ports.png)
+
+In this example, incoming traffic is allowed when it originates in the internal subnet `172.31.16.0/20`.
+
+## Generate a join token
+
+To add a machine to the appliance, you need a join token that provides the necessary information and authentication details to join the appliance.
+
+You must generate such a token on the existing machine that runs the Anbox Cloud Appliance. Use the following command:
+
+    anbox-cloud-appliance cluster add <name>
+
+The name you pick here is important because it will identify the node in both LXD and AMS.
```

Comment:
Maybe interesting for the user to add here that the first node is always `lxc0` so it makes sense to use `lxd<n>` to keep the same naming scheme

Reply:
Yes, `lxd0` is what I meant :-)